sklearn/semi_supervised/label_propagation.py

```diff
@@ -381,7 +402,10 @@ class LabelSpreading(BaseLabelPropagation):
       parameter for knn kernel
 
     alpha : float
-      clamping factor
+      Clamping factor [0, 1], it specifies the relative amount of the
```

Comment:
By a new `__init__` do you mean something like [this](https://github.com/scikit-learn/scikit-learn/pull/6727/commits/f6091051874f92674d609f79757f30fcbba3ab49)? 

What should be the default value for `alpha` in `BaseLabelPropagation.__init__`? What about `LabelPropagation.__init__`? The tests were failing because I ended up with `alpha=1` in `BaseLabelPropagation.__init__` and `alpha=None` in `LabelPropagation.__init__`, i.e., this constructor set a different default value for `alpha` than the former.